Add initial content to index page

namurphy opened the pull request:
My goals for this pull request are:

 - [ ] Add an intro paragraph that includes the dates of the hack week
 - [ ] Add a section on what a hack week is
 - [ ] Add a section that refers to the code of conduct

namurphy committed:
Add initial text for index page

StanczakDominik commented:
`main` here is the branch with the HTML output, and `src` is the nikola/markdown/ipynb source. I switched the default branch for the repo to `src`, so that this doesn't happen from now on.

namurphy commented:
> `main` here is the branch with the HTML output, and `src` is the nikola/markdown/ipynb source. I switched the default branch for the repo to `src`, so that this doesn't happen from now on.

Oh oops — thank you for the fix!

namurphy committed:
Add content to main page

namurphy committed:
Tentative remove feature cards from main page

namurphy committed:
Add a permanent invite link for Discord

There's still a chance that this invite link will be replaced, so
we'll need to watch it.

StanczakDominik reviewed (approved):
Definitely in the right direction, let's get it in! :)
